### ISSUE TITLE
[Duplicate] Consider publishing act to winget packages repository

### ISSUE THREAD

**Issue by rrmistry** [nektos/act#1489]
### Act version

act version 0.2.34

### Feature description

This is a duplicate of issue #253.

Just thought to create this issue to raise attention since the original issue was closed by a bot.

The blocker that the issue was waiting on (i.e. https://github.com/microsoft/winget-cli/issues/140) has since been resolved (i.e. https://github.com/microsoft/winget-cli/pull/2500).

Instructions on how to support `winget` seem to be found here [Authoring Manifests](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md).

But need some help to dig further.

CC: @rodrigoscna, @rodrigoscna, @chrpai

**Comment by cmjchrisjones** [comment]
I was just searching to see if Winget was available, 🤞 it gets considered :)

**Comment by sitiom** [comment]
Duplicate of #1146

**Comment by sitiom** [comment]
PR is merged. Please close this now.
